Makefile

```diff
@@ -503,6 +503,7 @@ clean-dist:
 .PHONY: clean
 
 publish: dist
+	export RELEASE_DOC_TAG=`echo ${RELEASE_TAG} | awk -F "\." '{print $1"."$2}'`
```

Comment:
I don't think this works in make, does it?

Reply:
You are correct, fixing